dyno-queues-redis/src/main/java/com/netflix/dyno/queues/redis/sharding/RoundRobinStrategy.java

```diff
@@ -0,0 +1,28 @@
+package com.netflix.dyno.queues.redis.sharding;
```

Comment:
License text missing

Reply:
does an idea behind this PR make sense for you ?

Reply:
Yes, it does :)